src/Mooc/User/Infrastructure/UserWriterRepositoryInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace CodelyTv\Mooc\User\Infrastructure;
+
+use CodelyTv\Mooc\User\Domain\User;
+
+interface UserWriterRepositoryInterface
```

Comment:
Hi, only one suggestion. Interface is an implementation detail and it is not necessary to add it. Just UserWriterRepository will be better.